script/testing/oltpbench/constants.py

```diff
@@ -14,6 +14,7 @@
 OLTP_DEFAULT_TERMINALS = 1
```

Comment:
Instead of the prefix `OLTP_` it should probably be `OLTPBENCH_`

Reply:
Done